packages/kas/src/__tests__/transforming.test.ts

```diff
@@ -9,29 +9,44 @@ expect.extend({
 
         return actual === expected
             ? {pass: true, message: () => ""}
-            : {pass: false, message: () => `${input} factors as ${reference}`};
+            : {
```

Comment:
These custom assertions were stating that the input matches the expected value when they failed, which was super confusing. 

For some, I was able to use Jest's internal utils for printing the expected and actual values as well as providing a nice "over/under" diff. 

Ref: https://dev.to/zirkelc/improve-your-testing-with-custom-matchers-in-jest-2ibm